modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,17 @@
+[[hostname-rename]]
+= Rename Hostname
+
+With the [command]``spacewalk-hostname-rename`` script it is possible to rename the hostname of the {productname} Server.
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+It is the best to proceed as follows: 
+
+. Adjust the hostname in advance.
+It is important to configure it locally, and for DNS resolution (including reverse resolution).
+. Load the new configuration.
+If in doubt whether the new configuration is running, reboot the server.
+. Run the script [command]``spacewalk-hostname-rename`` script with the public IP address of the server.
+. Some other actions are needed to make your landscape aware of the new hostname and IP address.
```

Comment:
I strongly recommend you check at the galaxy-devel mailing lists what other actions are required, so we can list them here.

As I sad at the card:

> NOTE: I only did basic testing, someone else with more experience with the script should help doc team.

Reply:
ok